sys/net/gnrc/network_layer/sixlowpan/iphc/gnrc_sixlowpan_iphc.c

```diff
@@ -246,15 +249,14 @@ static size_t _iphc_ipv6_decode(const uint8_t *iphc_hdr,
 
         case IPHC_SAC_SAM_16:
             ipv6_addr_set_link_local_prefix(&ipv6_hdr->src);
-            ipv6_hdr->src.u32[2] = byteorder_htonl(0x000000ff);
-            ipv6_hdr->src.u16[6] = byteorder_htons(0xfe00);
+            memcpy(&ipv6_hdr->src.u8[8], addr_fffe, sizeof(addr_fffe));
```

Comment:
`ipv6_hdr` was completely zeroed in l. 166 so it might make more sense to just set the non-zero values here and remove `addr_fffe` altogether:
```suggestion
            ipv6_hdr->src.u8[11] = 0xff;
            ipv6_hdr->src.u8[12] = 0xfe;
```